src/staticanalysis/bot/static_analysis_bot/report/phabricator.py

```diff
@@ -89,23 +94,40 @@ def publish_comment(self, revision, issues, patches):
         coverage_issues = [issue for issue in issues if issue.ANALYZER == COVERAGE]
 
         # First publish inlines as drafts
-        inlines = list(filter(None, [
-            self.comment_inline(revision, issue, existing_comments)
-            for issue in issues
-            if issue.ANALYZER not in ANALYZERS_WITHOUT_INLINES
-        ]))
-        if not inlines and not patches and not coverage_issues:
+        inlines_published = []
+        inlines_extras = []
+        for issue in issues:
+            if issue.ANALYZER in ANALYZERS_WITHOUT_INLINES:
+                continue
+
+            try:
+                self.comment_inline(revision, issue, existing_comments)
+                inlines_published.append(issue)
+            except ConduitError as e:
+                # Publish failures in top comment
+                # https://github.com/mozilla/release-services/issues/2028
+                logger.warn('Failed to publish inline comment for {}: {}'.format(issue, str(e)))
+                if self.publish_extra_issues:
+                    inlines_extras.append(issue)
+
+        if not inlines_published and not inlines_extras and not patches and not coverage_issues:
             logger.info('No new comments found, skipping Phabricator publication')
             return
-        logger.info('Added inline comments', ids=[i['id'] for i in inlines])
+        logger.info('Added {} inline comments'.format(inlines_published))
 
-        # Then publish top comment
-        non_coverage_issues = [issue for issue in issues if issue.ANALYZER != COVERAGE]
-        if len(non_coverage_issues):
+        # Then publish top comment about inline comments + clang-forant silent issues
```

Comment:
Is `clang-forant` a French Clang tool? :P

Reply:
:man_facepalming: